SIP-0065: Transfer of SOV from Adoption and Development Funds to Exchequer (#65)
DistributedCollective/SIPS: SIP-0065.md

---
SIP: '0065'

> Title: Permission update on the Adoption and Development Funds

Reviewer: ```suggestion
Title: Transfer of SOV from Adoption and Development Funds to Exchequer
```
